internal/store/horizontalpodautoscaler.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 The Kubernetes Authors All rights reserved.
+Copyright 2019 The Kubernetes Authors All rights reserved.
```

Comment:
Hmm not sure we need to change this or the file name itself 🤔Just to not lose the history for it. Thoughts @brancz @tariq1890

Reply:
It should stay 2016

Reply:
I follow @tariq1890 advice for this but it's ok now, I rollbacked the date.